tests/Unit/LoggerTraitTest.php

```
@@ -0,0 +1,230 @@
+    static::log('Test message');
+    static::logSection('Test', 'Test content');
+
+    // Set back to false.
+    static::loggerSetVerbose(FALSE);
+
+    // Verify methods are silent when verbose is false.
+    static::log('Silent message');
+    static::logSection('Silent', 'Silent content');
+
+    // Test completed successfully.
+    $this->assertTrue(TRUE);
```

PHPStan (2.1.15)

Call to method PHPUnit\Framework\Assert::assertTrue() with true will always evaluate to true.


(method.alreadyNarrowedType)